code/modules/projectiles/projectile/energy/thermal.dm

```diff
@@ -0,0 +1,43 @@
+/obj/projectile/energy/inferno
+	name = "molten nanite bullet"
+	icon_state = "infernoshot"
+	damage = 20
+	damage_type = BURN
+	flag = ENERGY
+	armour_penetration = 10
+	reflectable = NONE
+	wound_bonus = 0
+	bare_wound_bonus = 10
+	impact_effect_type = /obj/effect/temp_visual/impact_effect/red_laser
+
+/obj/projectile/energy/inferno/on_hit(atom/target, blocked, pierce_hit)
+	..()
+	if(iscarbon(target))
+		var/mob/living/carbon/cold_target = target
+		var/how_cold_is_target = cold_target.bodytemperature
+		if(how_cold_is_target < 100)
+			cold_target.Knockdown(100)
+			cold_target.apply_damage(20, BRUTE)
```

Comment:
this is either going to be incredibly degenerate or completely useless, with no in-between, depending on how well temperature guns and the like synergize with this. I'm not really sure why this effect is here, seeing as how neither of these projectiles affect the target's temperature?